src/lib/components/Table/Table.tsx

```diff
@@ -34,10 +35,30 @@ const TableComponent: FC<TableProps> = ({ children, striped, hoverable, classNam
   return (
     <div data-testid="table-element" className={classNames(theme.wrapper)}>
       <TableContext.Provider value={{ striped, hoverable }}>
+        <div className={classNames(theme.shadow, className)}></div>
         <table className={classNames(theme.base, className)} {...props}>
           {children}
         </table>
       </TableContext.Provider>
+      <style jsx>{`
```

Comment:
I don't think that we want to do this. You must use TailwindCSS classes. Flowbite is a TailwindCSS-based component library, there is no point to use CSS or SASS or whatever.

Reply:
https://tailwindcss.com/docs/utility-first